src/Common/UtilText/Filter/SeparatorToCamelCase.php

```diff
@@ -16,15 +16,17 @@ class SeparatorToCamelCase
      * @param string $separator
      * @param bool $upperCaseFirst
      *
+     * @phpstan-param non-empty-string $separator
+     *
      * @return string
      */
-    public function filter($string, $separator = '-', $upperCaseFirst = false): string
+    public function filter(string $string, string $separator = '-', bool $upperCaseFirst = false): string
```

Comment:
Does this method have unit tests?

Reply:
For now this exact method isn`t tested separately but it is tested as part of applying several manifests